sql/sql_yacc.yy

```diff
@@ -2982,6 +2982,12 @@ opt_ev_status:
             Lex->event_parse_data->status_changed= true;
             $$= 1;
           }
+        | ENABLE_SYM ON SLAVE
```

Comment:
Please read a discussion on [Jira|https://jira.mariadb.org/browse/MDEV-33782?focusedCommentId=289601&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-289601] about how to express in SQL way the desired effect.

This patch does address the FR, but it complicates earlier dubious design decision still.  As the FR requires to achieve on slave  `Event::status` as `ENABLE`  a straightforward way to help it is to encode such intent as a new option which would be merely for logging on master and unlogging on slave. That is it need not to affect/expand `Event::status`.

We could even leave the current name of `SLAVESIDE_ENABLED` but not regard it the way it's done in the current patch.
Also consider `WRITE_TO_BINLOG_ENABLE` (somewhat this naming would follow an existing `NO_WRITE_TO_BINLOG`) instead.

When it's specified ...